### PR TITLE
Added apiServerArguments.event-ttl to master-config

### PR DESCRIPTION
**Pull request by tpoitras**
- https://bugzilla.redhat.com/show_bug.cgi?id=1353139
- event-ttl has been made configurable with release 3.1

While I was adding this, I alphabetized the tables in Admin Solutions

### CODE REVIEW

**Comment by tpoitras** [comment]
Peer Review please @ahardin-rh @bfallonf @tnguyen-rh @adellape 
The diff / Files changed for this PR will be wacky because while I was adding content for this BZ, I alphabetized the tables in Admin Solutions.

Might be easier to review here:

http://file.bne.redhat.com/tpoitras/2016/eventttl/openshift-enterprise/event-ttl-BZ1353139/install_config/master_node_configuration.html#master-configuration-files

and here, where I've added it to the table of available options (under apiServerArguments):

http://file.bne.redhat.com/tpoitras/2016/eventttl/openshift-enterprise/event-ttl-BZ1353139/admin_solutions/master_node_config.html#master-configuration-file-options

Thank you! :santa:

File: admin_solutions/master_node_config.adoc [hunk 1]
```diff
@@ -192,45 +192,56 @@ and substitute in whatever values you want to change.
...
-  clientCA: ca.crt
-  keyFile: master.server.key
-  maxRequestsInFlight: 500
-  requestTimeoutSeconds: 3600
-----
-
-|`*corsAllowedOrigins*`
-|Specifies the host name to use to access the API server from a web application.
+|`*admissionConfig*`
+|Contains xref:../architecture/additional_concepts/admission_controllers.adoc#architecture-additional-concepts-admission-controllers[admission control plug-in] configuration. OpenShift has a configurable list of admission controller plug-ins that are triggered whenever API objects are created or modified. This option allows you to override the default list of plug-ins; for example, disabling some plug-ins, adding others, changing the ordering, and specifying configuration. Both the list of plug-ins and their configuration can be controlled from Ansible.
```
Comment (bfallonf) [inline admin_solutions/master_node_config.adoc]: "Contains _the_ admission control plug-in..."

s/OpenShift/{product-title}

File: admin_solutions/master_node_config.adoc [hunk 2]
```diff
@@ -192,45 +192,56 @@ and substitute in whatever values you want to change.
...
 
 |`*apiLevels*`
 |A list of API levels that should be enabled on startup; for example, `v1beta3` and `v1`.
 
-|`*masterPublicURL*`
-|The URL that clients use to access the OpenShift API server.
+|`*apiServerArguments*`
+a|Key value pairs that match the API server's command-line arguments and are passed directly to the Kubernetes API server. These are not migrated, but if you reference a value that does not exist, then the server will not start.
```
Comment (bfallonf) [inline admin_solutions/master_node_config.adoc]: The first sentense might be missing something at the start. "Contains" maybe?

File: admin_solutions/master_node_config.adoc [hunk 3]
```diff
@@ -265,6 +273,56 @@ etcdConfig:
...
+
+- `*MaxImagesBulkImportedPerRepository*` (integer): Controls the number of images that are imported when a user does a bulk import of a Docker repository. This number is set low to prevent users from importing large numbers of images accidentally. This can be set to `-1` for no limit.
+- `*DisableScheduledImport*` (boolean): Allows scheduled background import of images to be disabled.
+- `*ScheduledImageImportMinimumIntervalSeconds*` (integer): The minimum number of seconds that can elapse between when image streams scheduled for background import are checked against the upstream repository. The default value is `900` (15 minutes).
+- `*MaxScheduledImageImportsPerMinute*` (integer): The maximum number of image streams that can be imported in the background, per minute. The default value is `60`. This can be set to `-1` for unlimited imports.
+
+https://github.com/openshift/openshift-ansible/blob/master/inventory/byo/hosts.ose.example[This can be controlled with the Ansible inventory].
+
+|`*kubernetesMasterConfig*`
+|Contains information about how to connect to kubelet's KubernetesMasterConfig. If present, then start the kubernetes master in this process.
```
Comment (bfallonf) [inline admin_solutions/master_node_config.adoc]: s/in this process/with this process

File: admin_solutions/master_node_config.adoc [hunk 4]
```diff
@@ -417,24 +425,34 @@ and substitute in whatever values you want to change.
...
+
+|`*imageConfig*`
+|Holds options that describe how to build image names for system components.
+
+|`*iptablesSyncPeriod*`
+|(string) How often iptables rules are refreshed. This can be controlled with
+`*openshift_node_iptables_sync_period*` from the Ansible inventory.
+
+|`*kubeletArguments,omitempty*`
+|Key-value pairs that are passed directly to the Kubelet that matches the Kubelet's command line arguments. These are not migrated or validated, so if you use them, then they may become invalid. Use caution, because these values override other settings in the node configuration that may cause invalid configurations.
```
Comment (bfallonf) [inline admin_solutions/master_node_config.adoc]: s/matches/match

**Comment by bfallonf** [comment]
@tpoitras A few comments, but largely good :star:

File: admin_solutions/master_node_config.adoc (hunk 4 above)
Comment (tpoitras) [inline admin_solutions/master_node_config.adoc]: "passed directly to the Kubelet that match the Kublet's command line arguments" ??? I disagree. I'm leaving this as is, @bfallonf

**Comment by tpoitras** [comment]
Thanks @bfallonf 

Merging

**Pull request merged**